manager/db.go

```diff
@@ -1,6 +1,12 @@
 package manager
 
-import "github.com/rancher/catalog-service/model"
+import (
+	"fmt"
+
```

Comment:
No need for this blank line. Impors from the go standard lib should be in a single group